### PR TITLE
Feature/additional 1.1 unit tests

### PR DESCRIPTION
**Pull request by jlynema**

### CODE REVIEW

File: context.tests/ContextTestFiles/ContextSimple1.AdditionalElements [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+id: <2142848@direct.example.com>
+patient-id: 2.16.840.1.113883.19.999999:123456
+type: radiology/report
+purpose: research
+creation-time: 20210416080510.1245-4000
+patient: givenName=3DJohn; surname=3DDoe; middleName=3DJacob; dateOfBi=
+rth=3D1961-12-31; gender=3DM; postalCode=3D12345
+ihe-format-code=ihc: pcc/HL7+N0D/ADT_A01/2.5.1
+content-type: 79429-7/Admission notification note/urn:oid:2.16.840.1.1=
+13883.6.1/LOINC
```
Comment (JoeShook) [inline context.tests/ContextTestFiles/ContextSimple1.AdditionalElements:30]: The two lines noted are the considered the same line.  To enabled this the second line would start with a space.  I know we are just adding this extra metadata to test this does not break the 1.1 parser.  But we still need that space to keep the semantic meaning as described in section 3.0 Metadata: Content.

Comment (jlynema) [reply context.tests/ContextTestFiles/ContextSimple1.AdditionalElements:30]: Sure, I can update.   I've not seen this before, and was using line 26 and 27 as an example, and I didn't see a space there so I didn't look into the format

Comment (jlynema) [reply context.tests/ContextTestFiles/ContextSimple1.AdditionalElements:30]: Mimekit actually doesn't support the space as the first character to indicate a new line.

**Review by JoeShook — CHANGES_REQUESTED** [review]

**Review by JoeShook — APPROVED** [review]
Looks good but I will open an new issue to move away from MimeKit Header class for Metadata.

**Pull request merged**